kraken/litmus/common_litmus.py

```diff
@@ -0,0 +1,80 @@
+import kraken.invoke.command as runcommand
+import yaml
+import logging
+import time
+import sys
+
+
+def get_pod_ready_status():
+    pods_status = runcommand.invoke("kubectl get pods -n litmus -o yaml")
+    yaml_pod = yaml.full_load(pods_status)
+    logging.info('len of pod items' + str(len(yaml_pod['items'])))
+    for pod in yaml_pod['items']:
+        return pod['status']['phase']
+
+
+# Install litmus and wait until pod is running
+def install_litmus():
+    runcommand.invoke("kubectl apply -f "
+                      "https://litmuschaos.github.io/litmus/litmus-operator-v1.8.2.yaml")
+
+    # wait until pods are running
+
+    ready_status = get_pod_ready_status()
+    logging.info("first ready status for pod" + str(ready_status))
+    while ready_status != "Running":
+        time.sleep(10)
+        logging.info("ready status for pod" + str(ready_status))
+        ready_status = get_pod_ready_status()
+        if "Error" in ready_status:
+            logging.error("Litmus pod had error creating")
+            sys.exit(1)
+
+
+# Check status of experiment
+def check_experiment(engine_name, experiment_name, namespace):
+    chaos_engine = runcommand.invoke("kubectl describe chaosengines/%s -n %s | grep "
+                                     "Status" % (engine_name, namespace))
+    engine_status = chaos_engine.split(": ")[-1].strip()
+    while engine_status != "Running" and engine_status != "Completed":
+        time.sleep(10)
+        logging.info("Sleeping waiting for engine to start running")
+        chaos_engine = runcommand.invoke("kubectl describe chaosengines/%s -n %s | grep "
+                                         "Status" % (engine_name, namespace))
+        engine_status = chaos_engine.split(": ")[-1].strip()
+        # need to see if error in run
+        if "not found" in engine_status:
+            return False
+
+    if not chaos_engine:
+        return False
+    chaos_result = runcommand.invoke("kubectl describe chaosresult %s"
+                                     "-%s -n %s | grep Verdict" %
+                                     (engine_name, experiment_name, namespace))
+    status = chaos_result.split(": ")[1].strip()
+    while status == "Awaited":
+        logging.info("Waiting for chaos result to finish waiting, sleeping 10 seconds")
```

Comment:
Not a big deal, but this is chatty on the console for longer tests - get this repeatedly.    Maybe something to follow up on as we see how other Litmus scenarios run in kraken.